go-tools/honest spam/honest.go

```diff
@@ -0,0 +1,66 @@
+package main
```

Comment:
Probably makes sense to actually move this into the root folder under `/conf-rate-monitor` or something (with the correct adjusted module name)